src/proxy/src/test/java/ee/ria/xroad/proxy/testsuite/MessageTestCase.java

```diff
@@ -189,14 +189,14 @@ public String getDisabledNotice(ServiceId service) {
     /**
      * @return the server proxy HTTP handler
      */
-    public AbstractHandler getServerProxyHandler() {
+    public  Handler.Abstract getServerProxyHandler() {
```

Comment:
I have noticed this extra space in multiple places, maybe reformat bunch of files again?